Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -346,6 +346,24 @@ def bump_formula_pr
       EOS
     end
 
+    if new_url =~ %r{^https://github.com/([\w-]+)/([\w-]+)/archive/refs/tags/(v?[.0-9]*)\.tar}
+      owner = Regexp.last_match(1)
+      repo = Regexp.last_match(2)
+      tag = Regexp.last_match(3)
+      github_release_data = begin
+        GitHub::API.open_rest("#{GitHub::API_URL}/repos/#{owner}/#{repo}/releases/tags/#{tag}")
+      rescue GitHub::API::HTTPNotFoundError
+        nil
+      end
+
+      pr_message += <<~EOS if github_release_data.present?
+        <details>
+          <summary>#{github_release_data["prerelease"] ? "pre" : ""}release notes</summary>
+          #{github_release_data["body"]}
+        </details>
+      EOS
```

Comment:
```suggestion
      pr_message += if github_release_data.present?
        pre = "pre" if github_release_data["prerelease"].present?
        <<~XML 
          <details>
            <summary>#{pre}release notes</summary>
            #{github_release_data["body"]}
          </details>
        XML
```

Reply:
Does present? take the Boolean into account?

Reply:
@SMillerDev Yes, `present?` is truthy for `true`, a number, a non-empty String/Hash/Array, etc.